Microsoft.Toolkit.Win32/Microsoft.Toolkit.Wpf.UI.Controls/WindowsXamlHostBaseExt.cs

```diff
@@ -0,0 +1,62 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.ComponentModel;
+using System.Windows;
+using System.Windows.Data;
+using Microsoft.Toolkit.Win32.UI.XamlHost;
+using Microsoft.Toolkit.Wpf.UI.XamlHost;
+
+namespace Microsoft.Toolkit.Wpf.UI.Controls
+{
+    /// <inheritdoc />
+    public class WindowsXamlHostBaseExt : WindowsXamlHostBase
```

Comment:
Why can't this be merged with WindowsXamlHostBase? That is, what is the value of having another class here rather than having the behaviors here just "baked in" for everyone to use?

Reply:
My understanding has been that WindowsXamlHostBase is owned by a team that has some unusual requirements, some of which I don't fully understand, or even know about.   I created the Ext class to provide functionality I needed for the wrappers that still weren't in the Base.  Over time, the Ext class has gotten thinner, but my need for it still hasn't been fully dissolved.